application/src/main/java/org/opentripplanner/apis/gtfs/generated/GraphQLTypes.java

```diff
@@ -199,14 +200,10 @@ public static class GraphQLBicycleParkingPreferencesInput {
     public GraphQLBicycleParkingPreferencesInput(Map<String, Object> args) {
       if (args != null) {
         if (args.get("filters") != null) {
-          this.filters = ((List<Map<String, Object>>) args.get("filters")).stream()
```

Comment:
You have to run `yarn install` again before generating this code. The library version has changed: #6647